client/components/domains/register-domain-step/index.jsx

```diff
@@ -166,6 +166,19 @@ var RegisterDomainStep = React.createClass( {
 	},
 
 	searchForm: function() {
+		var placeholderText = this.translate( 'Enter a domain or keyword' ),
+			exampleDomains = '';
+
+		if ( ! this.props.isInSigup && abtest( 'domainSearchPlaceholderText' ) === 'searchForADomain' ) {
+			placeholderText = this.translate( 'Search for a domain' );
```

Comment:
This test is currently limited to users with an `en` locale, so I don't think we should translate this string or "e.g." below.